src/esw/brushed_motors/Controller.cpp

```diff
@@ -40,13 +39,39 @@ float Controller::getCurrentAngle() const {
     return currentAngle;
 }
 
+// REQUIRES: newAngleRad to be in radians
+// MODIFIES: currentAngle
+// EFFECTS: forces the angle of the controller to be a certain value
+void Controller::overrideCurrentAngle(float newAngleRad) {
+    auto ticks = (int32_t) (((newAngleRad) / (2 * M_PI)) * quadCPR); // convert to quad units
+
+    try {
+        makeLive();
+
+        uint8_t buffer[4];
+        memcpy(buffer, UINT8_POINTER_T(&ticks), sizeof(ticks));
+        I2C::transact(deviceAddress, motorIDRegMask | ADJUST_OP, ADJUST_WB,
+                      ADJUST_RB, buffer, nullptr);
```

Comment:
Yes, you are right, but it is handled in the Nucleo side. But good reminder, because it is currently not handled in the Nucleo side (I just changed it)